src/components/singleAccordion/SingleAccordion.unit.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { mount } from "@vue/test-utils";
```

Comment:
Nitpick: why `unit.spec.ts` rather than `spec.ts`? Are we expecting other types of tests?